packages/schema/lib/schemas/InputFieldsSchema.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const makeSchema = require('../utils/makeSchema');
+
+const InputFieldSchema = require('./InputFieldSchema');
+
+module.exports = makeSchema(
+  {
+    id: '/InputFieldsSchema',
+    description: 'An array or collection of input fields.',
+    type: 'array',
+    items: { $ref: InputFieldSchema.id },
```

Comment:
```suggestion
    items: {
      oneOf: [{ $ref: InputFieldSchema.id }, { $ref: FunctionSchema.id }],
    },
```

Currently, we can define input field with a function.

Reply:
Ahh thanks for the heads up!

Reply:
> instead of having to create these 2 different schemas for both inputField and outputField, renaming this module to DynamicInputFieldsSchema with that code change suggestion would do.

hmmm just to clarify, this means using `DynamicInputFieldsSchema` to be something like this? so both inputField and outputField can be represented? 
```
 items: {
    oneOf: [{ $ref: InputFieldSchema.id }, { $ref: OutputFieldSchema.id }, { $ref: FunctionSchema.id }],
  },
```

Reply:
No, that of outputField would be different but similar. Doing the above, that has both represented, would mean an inputField can specify a property unique to only outputField.

I am suggesting this for `DynamicInputFieldsSchema`

```
    items: {
      oneOf: [{ $ref: InputFieldSchema.id }, { $ref: FunctionSchema.id }],
    },
```

And this for `DynamicOutputFieldsSchema`

```
    items: {
      oneOf: [{ $ref: OutputFieldSchema.id }, { $ref: FunctionSchema.id }],
    },
```

Reply:
Ah that makes perfect sense, thank you!!